src/main/java/net/minecraftforge/fluids/ItemFluidContainer.java

```diff
@@ -3,10 +3,13 @@
 import net.minecraft.item.Item;
 import net.minecraft.item.ItemStack;
 import net.minecraft.nbt.NBTTagCompound;
+import net.minecraftforge.fluids.capability.templates.FluidTankItemStack;
 
 /**
  * Reference implementation of {@link IFluidContainerItem}. Use/extend this or implement your own.
+ * @deprecated See {@link net.minecraftforge.fluids.capability.ItemFluidContainer}
  */
+@Deprecated
```

Comment:
I'm pretty sure you could mkae ItemFluidContainer provide a wrapping capability automatically.